cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -562,7 +562,51 @@ impl Lockfile {
             .collect::<Result<Vec<_>, LockedManifestError>>()
     }
 
-    /// Produce a lockfile for a given manifest.
+    /// Merge included environments, resolve the merged manifest, and return the resulting lockfile
+    ///
+    /// Already resolved packages will not be re-resolved,
+    /// and already fetched includes will not be re-fetched.
+    pub async fn lock_manifest(
```

Comment:
You'll need to add a `Flox` parameter here so that you can check the feature flag.